operate/qa/integration-tests/src/test/java/io/camunda/operate/webapp/security/es/AuthenticationIT.java

```diff
@@ -118,14 +135,62 @@ public void setUp() {
   }
 
   @Test
-  public void shouldSetCookie() {
+  public void requestToInternalAPIShouldFailWithoutCSRF() {
```

Comment:
Can we move these tests into either a separate file or into the respective existing API test files? The Authentication tests are one of the ones that need overhaul in the refactoring and it would make my work easier if we can avoid adding to it for now. I don't mean don't test CSRF, but maybe do so in a different test suite.